src/import/load-sound.js

```diff
@@ -22,6 +22,13 @@ const loadSoundFromAsset = function (sound, soundAsset, runtime) {
         {data: soundAsset.data}
     )).then(soundId => {
         sound.soundId = soundId;
+        // Set the sound sample rate and sample count based on the
+        // the audio buffer from the audio engine since the sound
+        // gets resampled by the audio engine
+        const soundBuffer = runtime.udioEngine.getSoundBuffer(soundId);
```

Comment:
`udioEngine` ?

Reply:
oops! I fixed this locally but must have forgotten to push